.github/workflows/clang-tools-amd64.yml

```diff
@@ -88,7 +90,7 @@ jobs:
         curl -LO https://github.com/llvm/llvm-project/releases/download/llvmorg-${version}/cfe-${version}.src.tar.xz
         curl -LO https://github.com/llvm/llvm-project/releases/download/llvmorg-${version}/clang-tools-extra-${version}.src.tar.xz
     - name: get llvm-project
-      if: ${{ matrix.clang-version >= 9 }}
+      if: ${{ matrix.clang-version >= 9 || matrix.clang-version == '12.0.1' }}
```

Comment:
It's strange how the string `12.0.1` is apparently neither `>= 9` nor `< 7` (line 77). I would've expected it to be at least one of those things. Even more so, because `3.9 < 7` actually works.